template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/appsettings.json

```diff
@@ -2,33 +2,18 @@
 ////#if (OrganizationalAuth || IndividualAuth)
 //  "Microsoft": {
 //    "AspNetCore": {
-//      "Hosting": {
-//        "Kestrel": {
-//          "Endpoints": {
-//            "Localhost": {
-//              "Address": "127.0.0.1",
-//              "Port": "55555"
-//            },
-//            "LocalhostHttps": {
-//              "Address": "127.0.0.1",
-//              "Port": "43434",
-//              "Certificate": "HTTPS"
-//            }
-//          }
-//        }
-//      },
 //#if (IndividualLocalAuth)
```

Comment:
The issue with the balancing was fixed with #887, we should make sure that that change gets absorbed in this PR